operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -221,6 +223,18 @@ func (r *ReconcileIstioOperator) Reconcile(request reconcile.Request) (reconcile
 		return reconcile.Result{}, nil
 	}
 
+	if ir := iop.Annotations[IgnoreReconcileAnnotation]; ir == "true" {
+		scope.Infof("Ignoring the IstioOperator CR %s because it is annotated to be ignored for reconcile ", iopName)
+		return reconcile.Result{}, nil
+	}
+
+	// for backward compatibility, the previous applied installed-state CR does not have the ignore reconcile annotation
+	// TODO(richardwxn): remove this check and rely on annotation check only
```

Comment:
let's create an issue for this.

Reply:
https://github.com/istio/istio/issues/29394